src/NodeServices/IAnalyticsClient.cs

```diff
@@ -360,7 +360,15 @@ public enum Actions
         /// <summary>
         /// When a package conflict is encountered which involves at least one built-in package.
         /// </summary>
-        BuiltInPackageConflict, 
+        BuiltInPackageConflict,
+        /// <summary>
+        /// Sort event, when user wants to sort some information
+        /// </summary>
+        Sort,
+        /// <summary>
+        /// View Details event, when user wants to see the details from a resource
+        /// </summary>
+        ViewDetails,
```

Comment:
I think we can make it more generic by using `View` instead of `ViewDetails`.

Reply:
Done